tinyagentos/notes/shared_docs_store.py

```
@@ -160,6 +232,86 @@ async def delete_entry(self, entry_id: str) -> None:
+
+        Each dict contains: rev_index, editor_id, editor_type, op, created_at.
+        The diff and snapshot columns are intentionally omitted here; use
+        revision_diff / entry_text_at for those.
+        """
+        cur = await self._db.execute(
+            "SELECT rev_index, editor_id, editor_type, op, created_at "
+            "FROM shared_doc_entry_revisions WHERE entry_id = ? ORDER BY rev_index",
+            (entry_id,),
+        )
+        rows = await cur.fetchall()
+        return [dict(zip([c[0] for c in cur.description], r)) for r in rows]
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)